packages/front-b2b-link-rn/src/Front.tsx

```diff
@@ -55,10 +56,12 @@ const FrontFinance = (props: {
     ) {
       setShowWebView(false)
     }
-    if (type === 'brokerageAccountAccessToken') {
+    if (
+      (type === 'brokerageAccountAccessToken' && !isTransferLink) ||
+      type === 'transferFinished'
+    ) {
       setPayload(payload)
       props.onReceive?.(payload)
```

Comment:
`onReceive` type for `transferFinished` should be `TransferFinishedPayload`

Reply:
fixed